src/backend/joanie/core/models/customers.py

```diff
@@ -0,0 +1,20 @@
+"""
```

Comment:
not sure about the name of the file. Why not just `users.py`?

Reply:
I will define Company model in this file. So as you suggested `accounts.py` seems good to me.